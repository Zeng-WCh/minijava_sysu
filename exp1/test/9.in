(4/21*32-8*18)-13/(39-21+35+38-36*17-39*11+8)/11/(23*21-21)-24/(17+38-25)/35+22*(14*27-31)-33+30/(11-6-8)*6*28+(27+21-40+13-1-14)*11+15-(40/38+30)/12*(27/7*3*7/24+31*6/32)+20-(39*39-21*30+8+11)*31+(17-9*32/39-15*36)/35+(3*33-5*5+10/11*6/33)/30-33-(36/4/24+22-4*30/36)-35/11-(9/17*14/26/34-2+3)-12+(17-8-4)/10/1*2-(13/18+15)/23-16+(26*35*28-24-18*16-20/33)-38-(31+34*16-14/15-17+15)-9+(11+7+25-29)/6+(21-1*7+13*33+20+25)+30-(28+37+8-23-29+34)/8-14+1-(28*19/14)*3*(35*2-20+33-16-12/18+3)*36+(33*8/39)*21-30-(2-33*15)+10-(22/19*3+36+34+1+26)*20/32+(3/10*8*14)/25*(17/33+20*26+25-40)+26-16-(38/3/39*10/17-25)+19-32*37-18/31/(35-20+18-11-33+36)*19*(29+24*33)+6*(33+30*38-35*17-29-3)*28/(29/10*21+15*8-28+36/13*12-26)-31-(17*8*32/9+8-3*13-30+29/15)-25+(23-17+26-14*17-36/20*23)/17+(2*20+18-2/18)+20/(3+21/20-21+1)/6+(12+34+9/27+15+31/16*3/6*11)+29-9/17/13/(17-13-9)-13/(12*9*33+26+34-3/29+1)+39/24-18*34*25+2-(24+15*7-10+2-15*16*25)-34*36/(19-5-35+6-12)-37/(31*6/35*7-1+38/9-28-33/2)+28*35/8-(16*7/10-13)+11*(35+2-3/20-6/16/1*3*14)*39+29*35-2/15+12-(13/6/26)*1-(20*36-39-18+15+35/14*35-31*11)-1/(19+35*13+9-5)-40-6*13+35-10+(37-15/40*34/34*28/19+2/10)+11*35-29+17*(34/15+10/4*12+21)*22+(5-29-13/26)+34+29*(22+29*30+34/1+29+2-32)+18-35/7+(15+6-1)/25-(20/37+19-27)*26-37+10