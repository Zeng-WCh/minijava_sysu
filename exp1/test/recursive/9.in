0-5-4+6-2+3+9-1-8-4-5-3+9-9+5-5+3-1-8-2-8-6+8+9+8+5+2-0-6+8