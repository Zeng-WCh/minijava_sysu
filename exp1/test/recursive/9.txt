3-2-2+8+7-1+0+3-3-9-8-6+2+4-7-1-8+1+0+2-0-6-4+0+9-7+6-9-0-3