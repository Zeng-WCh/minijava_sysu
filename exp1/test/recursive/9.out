Input an infix expression and output its postfix notation:
05-4-6+2-3+9+1-8-4-5-3-9+9-5+5-3+1-8-2-8-6-8+9+8+5+2+0-6-8+
End of program.
